features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/extensions/LocaleExt.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.features.exploredash.ui.extensions
+
+import java.util.*
+
+val Locale.isMetric: Boolean
+    get() {
+        return !isO3Country.equals("usa", true) &&
+               !isO3Country.equals("mmr", true)
```

Comment:
I was curious about this list of non-metric countries.  When I was in England, I noticed that they are still using miles on distance on roads, though most of their measurements were metric.

![image](https://user-images.githubusercontent.com/5956129/145903660-5cc14414-4325-427c-b5e8-654cffbf6258.png)
![image](https://user-images.githubusercontent.com/5956129/145903849-a6a1f615-4ac1-41f7-84a6-354426edb43f.png)
https://en.wikipedia.org/wiki/Road_speed_limits_in_the_United_Kingdom

Also, Liberia is noted as not using the metric system in some older articles.

Reply:
Yes, I know of Britain's case - it seems to be very mixed there, officially metric but uses imperial a lot. I assumed it's acceptable to set any option for them as a default, but it seems to be that road distances are mostly imperial still. We can add them to this list until we see any complaints.
As for Liberia - yes, seems like it's also in the list of "special" countries that use imperial exclusively, I missed that. Will fix.

Reply:
Fixed